ui/src/app/pages/analysis/notebook-list.tsx

```diff
@@ -19,11 +20,24 @@ import {convertToResource} from 'app/utils/resources';
 import {ACTION_DISABLED_INVALID_BILLING} from 'app/utils/strings';
 import {WorkspacePermissionsUtil} from 'app/utils/workspace-permissions';
 import {BillingStatus, FileDetail, ResourceType} from 'generated/fetch';
+import {FlexColumn} from 'app/components/flex';
+import {faClock} from "@fortawesome/pro-regular-svg-icons";
 
 const styles = {
   heading: {
     color: colors.primary,
     fontSize: 20, fontWeight: 600, lineHeight: '24px'
+  },
+  cloneNotebookCard: {
+    backgroundColor: colorWithWhiteness(colors.disabled, 0.9),
+    minWidth: '200px', maxWidth: '200px',
```

Comment:
nit: we prefer one property per line, if it's multi-line

Reply:
Done